codebuild/release/validate.yml

```diff
@@ -21,7 +21,7 @@ phases:
   pre_build:
     commands:
       - cd examples
-      - sed -i "s/dynamodb-encryption-sdk/dynamodb-encryption-sdk==$VERSION/" test/requirements.txt
+      - sed -i "s/dynamodb-encryption-sdk/dynamodb-encryption-sdk==$VERSION/" ../../dev_requirements/test-requirements.txt
```

Comment:
Is this one too many `..`s? Looks like we've only `cd`'d one level in.

Can we run a test release to make sure this is all good?

Reply:
great catch! turns out i also needed to navigate up a folder in the `examples/tox.ini` file. I'll send the successful run over slack